packages/eas-cli/src/commands/metadata/lint.ts

```diff
@@ -0,0 +1,68 @@
+import { getConfig } from '@expo/config';
+import { Flags } from '@oclif/core';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+import { loadConfigAsync } from '../../metadata/config/resolve';
+import { createMetadataContextAsync } from '../../metadata/context';
+import { MetadataValidationError, logMetadataValidationError } from '../../metadata/errors';
+import { findProjectRootAsync } from '../../project/projectUtils';
```

Comment:
Maybe I'm missing something but I don't see this function in the import file

Reply:
Yeah, this was the previous API, before Will's changes.